src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,34 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todos } from '../../types/Todos';
```

Comment:
Consider renaming the 'Todos' type to 'Todo' since it represents a single todo item.